src/common/exceptions/http/base.http.exception.ts

```diff
@@ -9,7 +9,7 @@ export class BaseHttpException extends HttpException {
     public message: string,
     public statusCode: HttpStatus,
     public context: LogContext,
-    public code: AlkemioErrorStatus,
+    public code: AlkemioErrorStatus | string,
```

Comment:
why?

Reply:
This is my watch list to change to strongly typed, but the moment REST EP is a feature to be used while we develop rest of transition, and I foresee that I might need to need hacking on the way arbitrary strings there, while we are doing reset of transition in the system. Once we done, it will be reverted to strongly typed.